### ISSUE TITLE
Update duckdb driver

### ISSUE THREAD

**Issue by JCoder01** [dbeaver/cloudbeaver#2508]
**Describe the bug**
Sql Parser doesn't understand "CREATE SECRET" syntax
https://duckdb.org/docs/sql/statements/create_secret.html

**To Reproduce**
1: Create duckdb connection
execute
`CREATE SECRET (TYPE S3, REGION 'us-east-1', PROVIDER CREDENTIAL_CHAIN)`

ERROR:
SQL Error: java.sql.SQLException: Parser Error: syntax error at or near "SECRET"
LINE 1: CREATE SECRET (TYPE S3, REGION 'us-east-1', PR...

**Comment by LonwoLonwo** [comment]
Hello @JCoder01 

As I can see, secret creation is available only from DuckDB version 0.10.

https://duckdb.org/docs/sql/statements/create_secret.html

In CloudBeaver DuckDB version is 0.9.2

```
<dependencies>
        <dependency>
            <groupId>org.duckdb</groupId>
            <artifactId>duckdb_jdbc</artifactId>
            <version>0.9.2</version>
        </dependency>
</dependencies>
```

So, we need to upgrade the DuckDB driver.
Thanks for the report.

As a workaround for you for now - you can create a new DuckDB2 driver with correct library manually: https://github.com/dbeaver/cloudbeaver/wiki/Adding-new-database-drivers

**Comment by EvgeniaBzzz** [comment]
@JCoder01 DuckDB has been updated to version 0.10.2 in CloudBeaver 24.0.5.
Thank you for being with us 🦫
